### ISSUE TITLE
i device non compaiono

### ISSUE THREAD

**Issue by FabioHPT** [madmicio/lovelace-ld-2410-control-card#11]
Salve, ho un problema con la card... i devices non compaiono autromaticamente e se provo ad impostarli manualmente continua a dire che il device non è disponibile... dove sbaglio ?
Grazie

**Comment by madmicio** [comment]
È un problema di struttura del nome. Devi fare l'adoption bluetooth come riportato nella guida di Giovanni Castania. In questo modo le entità avranno il nome corretto.

**Comment by FabioHPT** [comment]
Solo bluetooth? Io ho configurato il rilevatore come wifi. Grazie

**Comment by magidel** [comment]
Anche io ho difficoltà nel far riconoscere il dispositivo...che anch'io ho configurato in WiFi.
Tra l'altro non è LD2410, ma LD2420. Che non sia compatibile con quest'ultima board?

**Comment by madmicio** [comment]
mi fate capire quele è il risultato dell'adoption. come è strutturato il nome di una qualsiasi entità?
2420 ancora non supportato dalla card

**Comment by magidel** [comment]
Ok attendo quando sarà compatibile con 2420 :)
Grazie

**Comment by markmghali** [comment]
Yes, I am also having issues. I have created my own ESP32 device using an ld2410. I want to be able to use your card in Home Assistant. Is that possible?
